### PR TITLE
some tarball's md5sum is not always fixed

### PR DESCRIPTION
**Pull request by michalliu**
some tarball's md5sum is not always fixed, we ignore it to avoid change it constantly

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -167,7 +167,7 @@ PKG_CONFIG_PATH="$TARGET_DIR/lib/pkgconfig" ./configure \
   --enable-libx264 \
   --enable-libx265 \
   --enable-nonfree
-PATH="$BIN_DIR:$PATH" make
+PATH="$BIN_DIR:$PATH" make -j`nproc`
```
Comment (zimbatm) [inline build.sh]: `nproc` is not available on my system

**Comment by zimbatm** [comment]
I think it would be better to use urls that return a fixed result instead. Potentially separate the urls and checksums in another file and have a script to update them.

**Comment by michalliu** [comment]
@zimbatm  not good, the goal of md5sum is make sure the downloaded file is correct. If we auto update the md5sum. The md5sum is meaningless.

File: build.sh (hunk 1 above)
Comment (michalliu) [inline build.sh]: fixed using a fallback method

**Comment by zimbatm** [comment]
Alright :)

**Pull request merged**